headhunter/demo/website_view.xml

```diff
@@ -0,0 +1,705 @@
+                                                                    <input type="hidden" class="form-control s_website_form_input" name="user_id" value="2"/>
+                                                                </div>
+                                                            </div>
+                                                        </div>
+                                                        <div class="mb-0 py-2 col-12 s_website_form_submit" data-name="Submit Button" style="">
+                                                            <div style="width: 200px;" class="s_website_form_label"/>
+                                                            <a href="#" role="button" class="btn btn-primary btn-lg s_website_form_send" data-bs-original-title="" title="">Submit</a>
+                                                            <span id="s_website_form_result" class="text-danger ml8"/>
+                                                        </div>
+                                                    </div>
+                                                </form>
+                                            </div>
+                                        </section>
+                                    </div>
+                                    <div class="col-lg-4 mt-4 mt-lg-0 o_colored_level" style="">
+                                        <ul class="list-unstyled mb-0 ps-2">
+                                            <li>My Company</li>
+                                            <li>
+                                                <i class="fa fa-map-marker fa-fw me-2"/>
+                                                <span class="o_force_ltr">3575 Fake Buena Vista Avenue</span>
+                                            </li>
+                                            <li>
+                                                <i class="fa fa-phone fa-fw me-2"/>
+                                                <span class="o_force_ltr">+1 (650) 555-0111</span>
+                                            </li>
+                                            <li>
+                                                <i class="fa fa-1x fa-fw fa-envelope me-2"/>
+                                                <span>info@yourcompany.example.com</span>
+                                            </li>
+                                        </ul>
+                                    </div>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                </t>
+            </t>
+        </field>
+    </record>
+    <record id="headhunter.contact_us_opportunity" model="ir.ui.view">
+        <field name="name">Contact Form (Opportunity)</field>
+        <field name="type">qweb</field>
+        <field name="key">website_crm.`1``</field>
+        <field name="inherit_id" ref="contact_us"/>
```

Comment:
Please update the view defined just above instead of inheriting from it

Reply:
ok